tests/coalaJSONTest.py

```diff
@@ -8,7 +8,8 @@
 
 from coalib import coala, coala_json
 from coala_utils.ContextManagers import prepare_file
-from tests.TestUtilities import bear_test_module, execute_coala
+from tests.TestUtilities import (
+    bear_test_module, execute_coala, TEST_BEARS_COUNT)
```

Comment:
split these into separate lines, with a trailing `,`
(in all files where you are using brackets)